db/re/item_combo_db.txt

```diff
@@ -805,6 +806,8 @@
 24249:24254,{ bonus2 bMagicAddRace,RC_All,1; bonus bMaxSPrate,1; if (getequiprefinerycnt(EQI_SHADOW_ACC_R)+getequiprefinerycnt(EQI_SHADOW_ACC_L)>=10) { bonus2 bMagicAddRace,RC_All,2; bonus bMaxSPrate,2; } }
 24253:28391,{ bonus2 bAddRace,RC_All,1; bonus bMaxHPrate,1; if (getequiprefinerycnt(EQI_SHADOW_ACC_R)+getequiprefinerycnt(EQI_SHADOW_ACC_L)>=10) { bonus2 bAddRace,RC_All,2; bonus bMaxHPrate,2; } }
 24255:28392,{ bonus2 bAddRace,RC_All,1; bonus bMaxHPrate,1; if (getequiprefinerycnt(EQI_SHADOW_ACC_R)+getequiprefinerycnt(EQI_SHADOW_ACC_L)>=10) { bonus2 bAddRace,RC_All,2; bonus bMaxHPrate,2; } }
+20701:2165,{ .@a = getequiprefinerycnt(EQI_GARMENT); .@b = getequiprefinerycnt(EQI_HAND_L); bonus2 bSubEle,Ele_Earth,.@b; bonus2 bSubRace,RC_Fish,.@a; }
```

Comment:
The combo increase the damage to earth monster by 1% per refine
`bonus2 bSubEle,Ele_Earth,.@b;` should be `bonus2 bAddEle,Ele_Earth,.@b;`

Reply:
For me IRO description match KRO description and the bonus should be
`bonus2 bAddEle,Ele_Earth,.@b; bonus2 bSubRace,RC_Fish,.@a;`

Reply:
Ok, I'll do it. :)